optimum/deepsparse/__init__.py

```diff
@@ -31,6 +31,18 @@
         "DeepSparseModelForSemanticSegmentation",
         "DeepSparseModelForTokenClassification",
     ],
+    "modeling_diffusion": [
+        "DeepSparseStableDiffusionXLImg2ImgPipeline",
+        "DeepSparseStableDiffusionXLPipeline",
+        "DeepSparseStabletableDiffusionXLPipelineBase",
```

Comment:
we still want to remove the XL pipelines, right?

Reply:
Right